src/common/execute.ts

```diff
@@ -24,7 +24,11 @@ export const executeAsync = async (
       await context.request(
         GraphqlActions.asyncSessionStatus,
         { sessionId },
-        { customEnvironment: DEFAULT_ENVIRONMENT_NAME },
+        {
+          customEnvironment: REQUEST_HEADER_IGNORED,
+          customWorkspaceId: REQUEST_HEADER_IGNORED,
+          customAuthorization: REQUEST_HEADER_IGNORED,
+        },
```

Comment:
I have some problem still

![image](https://user-images.githubusercontent.com/77967415/122950639-4a901600-d385-11eb-87e5-22b34952e87e.png)

Reply:
@eugeneantonevich so now it  error shows for any type login